contracts/UFragments.sol

```diff
@@ -98,42 +109,43 @@ contract UFragments is ERC20Detailed, Ownable {
         onlyMonetaryPolicy
         returns (uint256)
     {
+        uint256 oldTotalSupply = _totalSupply;
```

Comment:
consider letting the compiler do this optimization

Reply:
by using new version + optimizer

Reply:
https://github.com/ampleforth/uFragments/pull/197